apps/src/templates/progress/progressHelpers.js

```diff
@@ -71,10 +71,7 @@ export function stageLocked(levels) {
   // Given this, we should be able to look at the last level in our collection
   // to determine whether the LG (and thus the stage) should be considered locked.
   const level = levels[levels.length - 1];
-  return (
-    level.status === LevelStatus.locked ||
-    (level.kind === 'assessment' && level.status === 'submitted')
-  );
+  return level.locked;
```

Comment:
we're passing the result of this directly into a `PropTypes.bool.isRequired` field, so this needs to be `!!level.locked` to avoid prop validation warnings when `level.locked` is undefined.

Reply:
Just to circle back- I've updated this and the comment.